### ISSUE TITLE
Updating LilyPond detection for new binaries

### ISSUE THREAD

**Issue by Jean-Abou-Samra** [frescobaldi/frescobaldi#1429]
With [LilyPond 2.23.7](https://lists.gnu.org/archive/html/lilypond-user/2022-03/msg00342.html), the binaries are no longer distributed in the form of installers. They're just archives of static binaries that can be unpacked and run right away. As a corollary, LilyPond no longer provides an automechanism to install itself system-wide -- it is the choice of the user to symlink the executable as appropriate. It would be nice if there were an easier way for new users to set up Frescobaldi for using these binaries than adding a new LilyPond version and typing `/.../bin/lilypond` as the path.

**Comment by fedelibre** [comment]
#313 is about proposing the installation of LilyPond on first run

**Comment by fedelibre** [comment]
> It would be nice if there were an easier way for new users to set up Frescobaldi for using these binaries than adding a new LilyPond version and typing `/.../bin/lilypond` as the path.

An easy way might be a preference where you can choose a directory to scan to detect all the lilypond binaries (looking up for any file path ending with `bin/lilypond`).
The default directory might be where the lilypond binaries downloaded and installed by Frescobaldi (since version 3.3.0) are saved. For example, for the flatpak installation, an example path is: `/var/home/$USER/.var/app/org.frescobaldi.Frescobaldi/data/frescobaldi/frescobaldi/lilypond-binaries/lilypond-2.25.3/bin/lilypond`.

Basically, it's what's been suggested in #375 (sort of duplicate of this one).
